### PR TITLE
feat(flutterfire_ui_localizations): add Norwegian Translations

### PR DESCRIPTION
**Pull request by Maztermo**
## Description

Adds Norwegian (nb) as a locale for the flutterfire_ui_localization package

## Checklist

- [ x ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ x ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ x ] All existing and new tests are passing.
- [ x ] I updated/added relevant documentation (doc comments with `///`).
- [ x ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ x ] I read and followed the [Flutter Style Guide].
- [ x ] I signed the [CLA].
- [ x ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [ x ] No, this is *not* a breaking change.

### CODE REVIEW

File: packages/firebase_ui_localizations/lib/src/lang/nb.dart [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+  String get timestampLabel => "timestamp";
+
+  @override
+  String get typeLabel => "type";
+
+  @override
+  String get unknownError => "Det oppstod en ukjent feil";
+
+  @override
+  String get updateLabel => "update";
```
Comment (mhadaily) [inline packages/firebase_ui_localizations/lib/src/lang/nb.dart:242]: Tenker du vi bør endre dette til "Oppdater"?

**Comment by lesnitsky** [comment]
@Maztermo could you please run `melos run format` in your branch?

**Comment by Maztermo** [comment]
> @Maztermo could you please run `melos run format` in your branch?

Done 👍

**Comment by Maztermo** [comment]
@lesnitsky doesn't seem to me like the pub_dry_run failure is related to my fork, but let me know if I need to do anything else.

**Comment by lesnitsky** [comment]
Please open this PR in [firebase/FirebaseUI-Flutter](https://github.com/firebase/FirebaseUI-Flutter)

**Pull request closed without merging**